app/models/request_form/requester_address.rb

```diff
@@ -0,0 +1,25 @@
+module RequestForm
+  class RequesterAddress < Base
+    attribute :requester_proof_of_address
+    attribute :requester_proof_of_address_id
+    attr_accessor :default
+
+    validates :requester_proof_of_address, presence: true, unless: -> { Attachment.exists?(requester_proof_of_address_id) }
+    validates :requester_proof_of_address, file_size: { max: 7.megabytes }, file_type: { allowed: %w[image/jpg image/jpeg image/png application/pdf application/doc application/docx] }
```

Comment:
This needs to use the default options
```suggestion
    validates :requester_proof_of_address, file_size: { max: 7.megabytes }, file_type: true }
```